src/pages/homeCollection/kitAssembly.js

```diff
@@ -90,11 +96,21 @@ const kitAssemblyTemplate = async (name) => {
 
   document.getElementById("navbarNavAltMarkup").innerHTML = nonUserNavBar(name);
   contentBody.innerHTML = template;
+
+  // Set up automatic tabbing between inputs upon scanning (assuming the scanner automatically inputs the enter key at the end)
```

Comment:
It's duplicated in `assignKits.js`. Should we create a common function in `shared.js`.